e2e_tests/integration/types.spec.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2002-2018 "Neo4j, Inc"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* global Cypress, cy, test, expect */
+
+describe('Types in Browser', () => {
+  it('can connect', () => {
+    cy.executeCommand(':server disconnect')
+    cy.executeCommand(':clear')
+    cy.executeCommand(':server connect')
+    const password = Cypress.env('BROWSER_NEW_PASSWORD') || 'newpassword'
+    cy.connect(password)
+  })
+  it('presents the point type correctly', () => {
+    cy.executeCommand(':clear')
+    const query =
+      "WITH point({{}crs: 'wgs-84', longitude: 12.78, latitude: 56.7}) as p1 RETURN p1"
```

Comment:
Shouldn't `point({{}crs` be `point({crs` (without additional `{}`). I also see the tests pass with this approach.

Am I missing something as to why this passes?

Reply:
Try it :)
When you "type" things in cypress we have to escape "{" because it's a special character. And this is how you escape it.
In the tests you see that it types what we want it to type.